asv_runner/util.py

```diff
@@ -107,6 +107,8 @@ def human_float(value, significant=3, truncate_small=None, significant_zeros=Fal
     ("h", 60 * 60),
     ("d", 60 * 60 * 24),
     ("w", 60 * 60 * 24 * 7),
+    ("M", int(60 * 60 * 24 * 7 * 52 / 12)),
+    ("q", int(60 * 60 * 24 * 7 * 52 / 4)),
```

Comment:
Just as a stylistic nit, are the conversions for the other suffixes done elsewhere? It would be nice to be consistent perhaps.

Reply:
I don't think so, the place it's being used in the `asv` flow is here:
https://github.com/airspeed-velocity/asv/blob/b8cbf81ccb808158102692a6fc42b8a9b2f2861e/asv/util.py#L187
and there's not type guard there. Should there be?